CONTRIBUTING.md

```diff
@@ -0,0 +1,58 @@
+# PagesJS Contributing Guide
+
+On behalf of Yext, we'd like to thank you for your interest in PagesJS and appreciate any contributions you're willing to make! Please read through the following guide before making any pull requests.
+
+## Repo Setup
+
+The PagesJS repo is a monorepo using pnpm workspaces. The package manager used to install and link dependencies must be [pnpm](https://pnpm.io/).
+
+To develop and test:
+
+1. Run `pnpm i` in the root folder.
+
+2. Make your changes. Then in /packages/pages run `pnpm pack`.
+
+3. Reference the local pack file in another project that uses this one, such as https://github.com/yext/pages-starter-react-locations. You can reference it in the starter's package.json like `"@yext/pages": "file:../pages/packages/pages/yext-pages-1.0.0.tgz"` (assuming both projects live in the same root folder).
+
+4. If you make changes in PagesJS make sure to `rm -rf package-lock.json node_modules/ && npm i` in the starter first to ensure you pull in the latest packed changes.
```

Comment:
We can (and probably should) only remove node_modules/@yext/pages rather than the full folder

Reply:
good call, much faster to iterate on too